near-fetch/src/lib.rs

```diff
@@ -131,15 +147,17 @@ impl Client {
         signer: &dyn SignerExt,
         receiver_id: &AccountId,
         actions: Vec<Action>,
+        wait_until: Option<TxExecutionStatus>,
     ) -> Result<CryptoHash> {
         // Note, the cache key's public-key part can be different per retry loop. For instance,
         // KeyRotatingSigner rotates secret_key and public_key after each `Signer::sign` call.
         let cache_key = (signer.account_id().clone(), signer.public_key());
+        let wait_until_param = wait_until.unwrap_or(TxExecutionStatus::Included); // Default equal to legacy broadcast_tx_async
```

Comment:
This isn't actually true. It should be Execution Status of None. But when I do that, the response doesn't come back with a TX hash. Not sure how to rectify this.